src/controllers/dynakube/dtclient_reconciler.go

```diff
@@ -144,6 +144,15 @@ func (r *DynatraceClientReconciler) Reconcile(ctx context.Context, instance *dyn
 			}}
 	}
 
+	if instance.IsActiveGateMode(dynatracev1beta1.KubeMonCapability.DisplayName) &&
```

Comment:
this does not include the deprecated kubemon section, use `KubernetesMonitoringMode()` instead